modules/ROOT/pages/dataweave-type-reuse-modules.adoc

```diff
@@ -0,0 +1,38 @@
+= Reusing Types from DataWeave Modules
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Reuse types declared in other DataWeave modules by using the import directive:
+
+.DataWeave Module (`dw/Weather.dwl`):
+[source,dataweave,linenums]
+----
+%dw 2.0
+type Weather = "cloudy" | "sunny" | "rainy" | "stormy"
+type DetailedWeather = {temperature: Number, weather: Weather}
+----
+
+You can use those types as any other type in DataWeave, as constraints for variables and functions or for pattern matching and checking types at runtime:
```

Comment:
"those" -> imported?

I kind of want something before "as". "the same as"?

Reply:
Added imported.